### ISSUE TITLE
Timing Spoiled

### ISSUE THREAD

**Issue by Sasha79** [SubtitleEdit/subtitleedit#2467]
Thanks for great software!Also,I would like to make suggestions:

I didn't find this bugfix in Subtitle Edit Changelog(versions newer than mine):
-I have a problem with movies subtitle[like Back to the
future(1985)];sometimes I decide to

add soundtrack lyrics text in the middle of subtitle(usually,there is
no subtitle when song is

playing).When I save work in Subtitle Edit,the Start Time of subtitles
bellow this lyrics is

changed to start immediately after lyrics/new added text.Also,when an
incomplete

Subtitle.SRT is formed for some video mix(the subtitle does not
contain all the songs)and

after a while I get to add another extra song/text/lyrics/subtitle at
the beginning,or in the

middle,all the text below it(after it)changes the timing(it's start at
6:21 becomes 6:22,or

8:15 becomes 7:44),though I did not change that lower/older text,just
upper text(if there is

lyrics/text that starts in the tenth minute and I add text/lyrics
starts in the third minute and

ends up in the eighth minute,does he have to move a ten-minute song to
start immediately

after this in the eighth?)

I thought it is:
Subtitle Edit / GUI / Menu / Options / Settings / General / Min.
Duration Miliseconds: 2000;

Max Duration Miliseconds: 900000

or:

Subtitle Edit / GUI / Menu / Tools / Apply Duration Limits / Min
Duration Miliseconds: 2000;

Max Duration Miliseconds: 900000

[Also,there is no way to uncheck this option(Max Duraton)as opposed to
SW(Subtitle

Workshop)].Never mind the Max Duration is set to be 21000 ms,900000 ms
or 70000 ms,it changes itself to be 50000 ms or 188000 ms.Editing and
saved in Subtitle

Edit,Pet Shop Boys is set to start immediately after Breakfast Club at
07:44[and was

normally starting at 08:15(fortunately it was backed up)].
It did the Subtitle Edit;I also use SW,but SW is tested-it does not
make these changes
when I applied Extend To Next!.

-The third button in the GUI / Menu / File / Export / Export custom
text format is only half-

visible and it can not be resized.

I don't know if these are bugs or I missed something in Configuration.

Regards

Sasha

**Comment by darnn** [comment]
I don't speak for Nik, but:
a. Have you tried the latest beta? https://github.com/SubtitleEdit/subtitleedit/releases/download/3.5.3/SubtitleEditBeta.zip
b. I'm not even sure I understand correctly the first problem you're describing. You take an exisiting subtitle file, add a subtitle in the middle of it, set start and end times for it, and that changes the start/end times of the following subtitles? Because this doesn't happen for me when using "set start time" and "set end time" (F11 and F12). It might happen if you're using something like "set start and off-set the rest" (F9) accidentally. In the controls (the thing with the three tabs that read "translate", "create" and "adjust", make sure you're in the proper tab, because F9 does different things depending on which one you're in.

**Comment by Sasha79** [comment]
Thanks,it helped!
a.I use Subtitle Edit v3.5.3.0 from May 12,2017
b.You were correct:I take existing subtitle file,add a subtitle in the middle of it, set start time for it and
that changes the start times of the following subtitles.That was happening when I used "set start and off-set the rest" (F9) under tab "adjust".
F11 does exactly what I want!
You know,I searched for solution for that function in Subtitle Edit help file before... 
After I read your comment,I still don't find explanation in Subtitle Edit help file,that's why I didn't use it correctly.Function of F11 is not very clear explained.
